forc-plugins/forc-doc/src/render.rs

```diff
@@ -65,7 +64,7 @@ impl RenderedDocumentation {
                 file_contents: HTMLString::from(doc.clone().render(render_plan.clone())?),
             });
             // Here we gather all of the `doc_links` based on which module they belong to.
```

Comment:
Does this comment need an update?

Reply:
It's still accurate, it was just condensed into a function